### DISCUSSION TITLE
Setups/Screenshots

### DISCUSSION THREAD

**Discussion by cmacrae** [cmacrae/spacebar#65] [Show and tell]
Let's see what people are doing with their configurations 💅  
Share a screenshot of your desktop/bar/sections that you're proud of! 📷  

Feel free to add any context/details underneath :)

**Comment by cmacrae** [comment]
## Full (not so full...)
![image](https://user-images.githubusercontent.com/3392199/114203587-35901300-9950-11eb-9ed3-25a7ea55860f.png)
Just lonely Emacs

## Left side
![image](https://user-images.githubusercontent.com/3392199/114203769-5d7f7680-9950-11eb-8c4d-ad5a3d7c3a52.png)
I've been using a pretty minimal setup whilst developing some features for v1.3.0.  
On the left is the space indicator using the new (soon to be released) "spaces for all displays" set of options, where space 2 is the focused space on my primary display and space 3 is the focused space on my secondary display

## Right side
![image](https://user-images.githubusercontent.com/3392199/114203784-61ab9400-9950-11eb-87df-ddf318ecd763.png)
And here on the right side, I'm using the (yet again, soon to be released) `right_shell_*` features to display my unread e-mail count (I know! I need to catch up on some e-mail 😬 )

Feel free to ask any questions!

**Reply by timrockx** [reply]
How did you get all of the icons to work? I tried the unicode and glyphs from fontawesome but I end up just getting the replacement character.

**Reply by dirmeier** [reply]
This looks so nice! One question: how did you remove the window decorations on MacOSX?

**Reply by 10blackhole** [reply]
You can remove the window decorations on MacOS in System Preferences!

**Comment by knowler** [comment]
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/6908001/115272409-fc039880-a103-11eb-998c-6e5f39783853.png">

I'm using a bit of a mishmash of colours right now between spacebar, my desktop, Neovim and its plugins, and Alacritty. I like a fuller formatted date including the timezone since I always forget it and I find it helpful to reference since I work on a remote team. I used to be more organized as far as spaces go, naming them and adding icons, but these days I usually have about 3 or 4 which are organized however I happen to at the start of my work day (i.e. usually code, preview, chats and music).

I was playing around with the new shell feature which I was using to display my tracked time for the day (i.e. `wakatime --today` — I might actually need to report a bug, I just haven't gotten around to reproducing it and documenting it). Looking forward to the new features. 😃

**Reply by notflip** [reply]
That's a beautiful terminal! Mind sharing some of your config? I'm pretty new to mac terminal setups, used to use Iterm2 but I want to switch to Kitty or Alacritty

**Reply by Nosp1** [reply]
Agreed,  I'd also be interested in your config!

**Comment by Ducky6944** [comment]
Right Side:
<img width="574" alt="Screen Shot 2021-07-21 at 2 34 54 PM" src="https://user-images.githubusercontent.com/12100960/126541559-5e1f2102-058e-468f-94cc-677a0628da4f.png">
<img width="587" alt="Screen Shot 2021-07-21 at 2 34 45 PM" src="https://user-images.githubusercontent.com/12100960/126541562-dae9e7c6-6dc2-4ec1-969a-e19767202a75.png">
<img width="589" alt="Screen Shot 2021-07-21 at 2 34 41 PM" src="https://user-images.githubusercontent.com/12100960/126541564-a5269c1d-75c8-437c-9de8-29e598c6a0d8.png">
Left Side:
<img width="431" alt="Screen Shot 2021-07-21 at 2 35 50 PM" src="https://user-images.githubusercontent.com/12100960/126541619-57e884b7-b777-44ba-991e-08d4602c300b.png">
Desktop:
<img width="1680" alt="Screen Shot 2021-07-21 at 2 33 37 PM" src="https://user-images.githubusercontent.com/12100960/126541565-55c11a54-a212-4eee-87e4-8717c8c57bea.png">

I have a while script which displays various information on the right side of my spacebar. Looking forward to some of the proposed updates.

**Reply by Meaticus22** [reply]
What wm are you using here? Assuming yabai?

**Comment by 10blackhole** [comment]
How can do to set icons. In my spacebarrc only displays a box :(

**Reply by 10blackhole** [reply]
<img width="1440" alt="Captura de pantalla 2022-12-16 a la(s) 23 31 14" src="https://user-images.githubusercontent.com/104014081/208219536-506780b5-23af-4f37-9ba3-0a08cc2173f7.png">

**Reply by bashu** [reply]
use font, that have icons, see Nerd Fonts. I use Hack Nerd Font for spacebar
```shell
spacebar -m config status_bar_text_font         "Hack Nerd Font:Bold:12.0"
spacebar -m config status_bar_icon_font         "Hack Nerd Font:Regular:12.0"
```

**Reply by 10blackhole** [reply]
My spacebarrc looks like that, but the icons do not display. How can i add them?

```
!/usr/bin/env sh
spacebar -m config display main
spacebar -m config position top 
spacebar -m config height 26
spacebar -m config title off
spacebar -m config spaces on
spacebar -m config space_icon_strip 1 2 3 4 5 6 7 8
spacebar -m config spaces_for_all_displays on
spacebar -m config space_icon_color red
spacebar -m config display_separator on
spacebar -m config display_separator_icon |
spacebar -m config display_separator_icon_color white
spacebar -m config clock on
spacebar -m config clock_icon ?
spacebar -m config power on
spacebar -m config clock_format      "%A, %B %-d, %Y %R %Z"
spacebar -m config dnd on
spacebar -m config status_bar_text_font         "Hack Nerd Font:Bold:12.0"
spacebar -m config status_bar_icon_font         "Hack Nerd Font:Regular:12.0"
```

**Reply by bashu** [reply]
you have ? in your file, instead of 

Can you run this command in terminal
```sh
spacebar -m config clock_icon                   
```

**Reply by bashu** [reply]
Make sure your editor doesn't strip away non-ascii symbols

**Reply by 10blackhole** [reply]
When i put thus in terminal, some icons appear on the bar. Furthermore, if i do this in my spacebarrc, the bar shows a ?. :(

**Reply by 10blackhole** [reply]
<img width="715" alt="Captura de pantalla 2022-12-17 a la(s) 10 28 20" src="https://user-images.githubusercontent.com/104014081/208244319-455fefc6-100d-437f-a055-c2a93479ee88.png">

**Reply by bashu** [reply]
share the config

**Reply by 10blackhole** [reply]
<img width="778" alt="Captura de pantalla 2022-12-17 a la(s) 11 06 09" src="https://user-images.githubusercontent.com/104014081/208245899-54c6b4fb-f337-464d-aab9-0cfe2d0fce24.png">

**Reply by 10blackhole** [reply]
```
`!/usr/bin/env sh

spacebar -m config status_bar_text_font         "Hack Nerd Font:Bold:12.0"
spacebar -m config status_bar_icon_font         "Hack Nerd Font:Regular:12.0"
spacebar -m config display main
spacebar -m config position top 
spacebar -m config height 26
spacebar -m config title off
spacebar -m config spaces on
spacebar -m config space_icon_strip 1 2 3 4 5 6 7 8
spacebar -m config spaces_for_all_displays on
spacebar -m config space_icon_color red
spacebar -m config display_separator on
spacebar -m config display_separator_icon |
spacebar -m config display_separator_icon_color white
spacebar -m config clock on
spacebar -m config clock_icon ÔóØ
spacebar -m config power on
spacebar -m config power_icon_strip            ÔâÄ   Ôñà    
spacebar -m config clock_format      "%A, %B %-d, %Y %R %Z"
spacebar -m config dnd on
```

**Reply by bashu** [reply]
fix your text editor

**Reply by 10blackhole** [reply]
Why? It shows the icons. Is problem of the editor?

**Reply by bashu** [reply]
this doesn't looks like an icons

![Screenshot 2022-12-17 at 8 36 55 PM](https://user-images.githubusercontent.com/144783/208247206-caddff81-e628-4997-a56e-89e6952148f0.png)

**Reply by bashu** [reply]
> Why? It shows the icons. Is problem of the editor?

yes, it saves garbage in a config file

**Reply by 10blackhole** [reply]
Before this pic i had uploaded a screenshot of the spacebarrc on my editor.

**Reply by bashu** [reply]
well, if I copy & paste what you have sent, I will get squares with question mark instead of icons. my work is done here, good luck

**Reply by 10blackhole** [reply]
It occurs when i copy the text on github, on my editor it display the icons.

**Reply by bashu** [reply]
If I copy something from my emacs and paste it here, it can be copied by someone and used. All I got from you has textual garbage instead of icons.

**Reply by 10blackhole** [reply]
I have solved the problem. Thanks so much :)

**Reply by bashu** [reply]
what was that? not saving file in unicode?

**Reply by 10blackhole** [reply]
I think i was not installed the fonts required xd

**Reply by bashu** [reply]
Now you can post your rice / setup

**Reply by 10blackhole** [reply]
i'm on Mac :(

**Comment by Meaticus22** [comment]
### Left side
<img width="511" alt="image" src="https://github.com/cmacrae/spacebar/assets/117058929/5e20bd7e-134d-4f25-97e1-0564d1e7be16">

### Right side
<img width="723" alt="image" src="https://github.com/cmacrae/spacebar/assets/117058929/55d6eac1-486b-422b-a6d7-983751d591e4">

### Desktop
<img width="1920" alt="image" src="https://github.com/cmacrae/spacebar/assets/117058929/744b03a5-015b-41dd-bdae-909fbb529f06">

Dotfiles: https://github.com/Meaticus22/dotfiles
